cypress/support/cypress-support/src/utils.js

```diff
@@ -205,9 +205,11 @@ function getTopic(
     topic = deviceMac + '_' + getEnvVariable(CONSTANTS.THIRD_PARTY_APP_ID);
   }
   if (operation == CONSTANTS.SUBSCRIBE) {
-    return subscribeSuffix ? topic + subscribeSuffix : topic + CONSTANTS.TOPIC_SUBSCRIBE_SUFFIX;
+    return subscribeSuffix
+      ? topic + subscribeSuffix
+      : topic + getEnvVariable(CONSTANTS.PUB_SUB_SUBSCRIBE_SUFFIX);
   } else {
-    return topic + CONSTANTS.TOPIC_PUBLISH_SUFFIX;
+    return topic + getEnvVariable(CONSTANTS.PUB_SUB_PUBLISH_SUFFIX);
   }
```

Comment:
will this handle if config module does not have these env values ? I believe the ask was to have a default env inside FCS for these ?

Reply:
yes, added default env inside cypress.config.js. Apologies for not adding the changes in the PR before.